userRoutes.js

```diff
@@ -0,0 +1,53 @@
+const router = require('express').Router();
+const User = require('./userModel');
+
+//GET USERS
+router.get('/users', async (req,res)=> {
+    const user = await User.find();
```

Comment:
Could rename "user" to "users", since we are trying to get all of the users in this endppoint